flux/index.js

```diff
@@ -3,11 +3,13 @@ import createLogger from 'redux-logger'
 import reduxPromise from 'redux-promise'
 import reduxThunk from 'redux-thunk'
 import {AsyncStorage} from 'react-native'
+import {allViews} from '../views/views'
 
 export const SAVE_HOMESCREEN_ORDER = 'SAVE_HOMESCREEN_ORDER'
+let defaultViewOrder = allViews.map(v > v.view)
```

Comment:
Ah. My typo, then. Apologies.

You'll need `=>`, instead of `>`.

This is currently doing `allViews.map(false)`, essentially, because we've got a boolean comparison instead of a lambda function.